data/en/directorylist.json

```diff
@@ -71,7 +71,7 @@
 	"engines": {
 		"coldfusion": {
 			"minimum_version": "9",
-			"notes": "Only CF11+ supports the type argument.",
+			"notes": "Only CF11+ supports the type argument. CF2018 added named parameters.",
```

Comment:
CF2018 introduced named parameters for all functions, so I don't think it's necessary for the notes here.

Reply:
Changes made.  CF2018 comment removed.

Reply:
I don't think this was removed.

Reply:
Updated. Check now. Not sure what happened there.